### PR TITLE
Add pointer to repository for userspace regression

### PR DESCRIPTION
**Pull request by DrMullings**
Now the reviewer gets a link where to look for the errors

- Related ticket: https://progress.opensuse.org/issues/34669
- Verification run: http://pinky.arch.suse.de/tests/1273#

### CODE REVIEW

**Comment by DrMullings** [comment]
@okurz @Soulofdestiny @rwx788

File: tests/qa_automation/install_test_suite.pm [hunk 1]
```diff
@@ -17,6 +17,7 @@ use testapi;
 
 sub run {
     my $test = get_required_var('QA_TESTSUITE');
+    record_info("The corresponding repository can be found at http://build.suse.de/package/show/QA:HEAD/$test");
```
Comment (rwx788) [inline tests/qa_automation/install_test_suite.pm]: I would also put link to the repo source.

**Comment by DrMullings** [comment]
Updated

**Comment by okurz** [comment]
Very simple and nice, thank you so much!

**Pull request merged**